packages/apps/app-dashboard/src/components/user-dashboard/consent/ui/PolicyForm.tsx

```diff
@@ -70,6 +70,10 @@ export const PolicyForm = forwardRef<PolicyFormRef, PolicyFormProps>(
       <div className="mt-4 p-4 bg-opacity-50 rounded-lg border border-opacity-20">
         <style>
           {`
+            .policy-form-${policy.ipfsCid} * {
+              color: ${isDark ? 'rgb(255 255 255 / 0.8)' : 'rgb(17 24 39)'} !important;
```

Comment:
All of the branches sprinkled around our CSS logic based on `isDark` should be handled using built-in tailwind theme functionality. Please create a ticket to update theme handling to use top-level css instead of making every component aware of the theme